johnsonandjohnson/Bodiless-JS
# feat(release, publish): Automate package publication on GitHub.

## Changes

When a new version tag is committed to any branch in GitHub, new package versions are published to NPM.

## Test Instructions


- Clone developer's repo on local.
- Checkout dev branch with related changes.
- Switch to master and merge in dev branch updates (publish limited to master branch). 
- Update packages and point to a different npm package scope.
- Setup travis CI by connecting to tester's github account.
- Add 'NPM_TOKEN' environment variable on "https://travis-ci.org/github/[tester account name]/Bodiless-JS/settings" page
- Commit some changes on packages.
- Run `npm run publish patch`

Expect
- Packages published to new scope/org.

## Related Issues

Fixes #16
